examples/medplum-mso-demo/README.md

```diff
@@ -0,0 +1,103 @@
+<h1 align="center">Medplum Managed Service Organization (MSO) Demo</h1>
+<p align="center">A starter application for building a multi-tenant healthcare platform on Medplum.</p>
```

Comment:
Can you add 1-2 sentences more about what we mean by 'multi-tenant' platform here. Want the reader to be able to identify "does this example apply to my application"?